### PR TITLE
feat (Typescript): Add typescript support , work done by @Metavirulen with PR fixes.

### PR DESCRIPTION
**Pull request by jafin**
This is #24 redone with PR comments.

### CODE REVIEW

**Comment by shimil** [comment]
Hey @jafin, is there any work being done on this PR?

**Comment by dan-kez** [comment]
Is there anything else for this PR to be added? It would be great to have typings.

**Comment by dan-kez** [comment]
@huyvohcmc and @TienNguyenEH and @tvkhoa (adding you all as you have recently approved a PR)

Can we merge this? We presently have to pin to @jafin 's branch to have TS support

**Comment by subs1da** [comment]
Are there any plans to merge this? Would be great... 🙏

File: package.json [hunk 1]
```diff
@@ -47,6 +47,8 @@
     "start": "react-scripts start"
   },
   "dependencies": {
-    "popper.js": "^1.11.1"
-  }
-}
+    "popper.js": "^1.11.1",
+    "@types/react": "*"
```
Comment (moshfeu) [inline package.json]: Is this should be in `devDependencies`?

Comment (jafin) [reply package.json]: @moshfeu yep, fixed. pushed to PR.

Comment (moshfeu) [reply package.json]: I hope that @tvkhoa will ever merge it 🙏

**Comment by tvkhoa** [comment]
Tested. It works. Thank you @jafin

**Pull request merged**